src/redis-pubsub.ts

```diff
@@ -11,10 +10,21 @@ export interface PubSubRedisOptions {
 
 export class RedisPubSub implements PubSubEngine {
 
-  constructor(options: PubSubRedisOptions = {}) {
+  constructor(options: PubSubRedisOptions = {}, publisher?: RedisClient, subscriber?: RedisClient) {
```

Comment:
Accidentally commented on the commit...
Could you please move the publisher and subscriber clients into the `PubSubRedisOptions` object?